Code/Editor/Util/3DConnexionDriver.cpp

```diff
@@ -60,34 +60,29 @@ bool C3DConnexionDriver::InitDevice()
         {
             UINT nchars = 300;
             TCHAR deviceName[300];
-            if (GetRawInputDeviceInfo(m_pRawInputDeviceList[i].hDevice,
-                    RIDI_DEVICENAME, deviceName, &nchars) >= 0)
-            {
-                //_RPT3(_CRT_WARN, "Device[%d]: handle=0x%x name = %S\n", i, g_pRawInputDeviceList[i].hDevice, deviceName);
-            }
+            GetRawInputDeviceInfo(m_pRawInputDeviceList[i].hDevice,
+                RIDI_DEVICENAME, deviceName, &nchars);
```

Comment:
This statement can be deleted since the `deviceName` variable isn't used after this.